e2e/rbd.go

```diff
@@ -568,6 +568,40 @@ var _ = Describe("RBD", func() {
 				createRBDStorageClass(f.ClientSet, f, nil, nil)
 			})
 
+			// Mount pvc to pod with invalid mount option,expected that
+			// mounting will fail
+			By("Mount pvc to pod with invalid mount option", func() {
+				deleteResource(rbdExamplePath + "storageclass.yaml")
+				createRBDStorageClass(f.ClientSet, f, map[string]string{rbdmountOptions: "debug,invalidOption"}, nil)
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					Fail(err.Error())
+				}
+				pvc.Namespace = f.UniqueName
+
+				app, err := loadApp(appPath)
+				if err != nil {
+					Fail(err.Error())
+				}
+				app.Namespace = f.UniqueName
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					Fail(err.Error())
+				}
+				// create an app and wait for 2 min for it to go to running state
+				err = createApp(f.ClientSet, app, 2)
```

Comment:
just wondering how we arrive on this timeout of `2` mins and is this enough? or will it cause the e2e goes wrong if the creation takes more time ?

Reply:
never seen an issue with this number, waiting for more time increases the time taken to execute E2E. if we face any issue this can be changed later also